.github/workflows/image.yaml

```diff
@@ -16,11 +16,17 @@
 name: image
 
 on:
-  push:
-    branches:
-      - "pull-request/[0-9]+"
-      - main
-      - release-*
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        type: string
+      build_multi_arch_images:
+        required: true
+        type: string
+      push_on_build:
+        required: true
+        type: string
```

Comment:
This is always `true`. Could we remove it?

Reply:
yes